spec/bolt/validator_spec.rb

```diff
@@ -134,6 +134,83 @@ def validate
         /Value at 'option' is a plugin reference, which is unsupported at this location/
       )
     end
+
+    context 'with suboptions' do
```

Comment:
Can we test this with the actual options? `plugins` doesn't allow `_plugin`, but any suboptions of `plugins` do. And then we can use the cases that raised this (i.e. key-data) to verify that defaults work.

Reply:
Added integration tests for `plugins`, `plugins.*`, `private-key.key-data`, `interpreters`, and `interpreters.*`. That should cover `_plugin` inheritance from parent options and the cases that raised this issue.

I'm wondering if we should just explicitly test each option or not...

Reply:
I think general testing is good, we can manually test each option but don't want to do that for every spec run.